src/pkg/logger/logger.go

```diff
@@ -65,58 +78,102 @@ func defaultLogLocation() string {
 // need to parse the log level before we execute the command.
 func AddLoggingFlags(cmd *cobra.Command) {
 	fs := cmd.PersistentFlags()
-	fs.StringVar(&llFlag, logLevelFN, "info", "set the log level to debug|info|warn|error")
 
-	// The default provided here is only for help info
-	fs.StringVar(&lfFlag, logFileFN, "corso-<timestamp>.log", "location for writing logs, use '-' for stdout")
+	addFlags(fs, "corso-<timestamp>.log")
+
+	//nolint:errcheck
+	fs.MarkHidden(ReadableLogsFN)
+	// TODO(keepers): unhide when we have sufficient/complete coverage of PII handling
+	//nolint:errcheck
+	fs.MarkHidden(SensitiveInfoFN)
+}
 
-	fs.Bool(debugAPIFN, false, "add non-2xx request/response errors to logging")
+// internal deduplication for adding flags
+func addFlags(fs *pflag.FlagSet, defaultFile string) {
+	fs.StringVar(
+		&LogLevelFV,
+		LogLevelFN,
+		LLInfo,
+		fmt.Sprintf("set the log level to %s|%s|%s|%s", LLDebug, LLInfo, LLWarn, LLError))
+
+	// The default provided here is only for help info
+	fs.StringVar(&logFileFV, LogFileFN, defaultFile, "location for writing logs, use '-' for stdout")
+	fs.BoolVar(&DebugAPIFV, DebugAPIFN, false, "add non-2xx request/response errors to logging")
 
-	fs.Bool(
-		readableLogsFN, false,
+	fs.BoolVar(
+		&ReadableLogsFV,
+		ReadableLogsFN,
+		false,
 		"minimizes log output for console readability: removes the file and date, colors the level")
-	//nolint:errcheck
-	fs.MarkHidden(readableLogsFN)
+
+	fs.StringVar(
+		&SensitiveInfoFV,
+		SensitiveInfoFN,
+		PIIHash,
```

Comment:
@ryanfkeepers - let's set the default to `PIIPlainText` for CLI. Let users change this if needed.

Reply:
Done